src/test/feefrac_tests.cpp

```diff
@@ -17,6 +17,44 @@ BOOST_AUTO_TEST_CASE(feefrac_operators)
     FeeFrac empty{0, 0};
     [[maybe_unused]] FeeFrac zero_fee{0, 1}; // zero-fee allowed
```

Comment:
This is [always used now](https://github.com/bitcoin/bitcoin/commit/0606e66fdbb914f984433d8950f0c32b5fb871f3#diff-eca436ae09a12f8a59ac888c7f4ee6f36e4d12f38c2dab47fee0377ce69fe660R18), right?